app/code/core/Mage/Wishlist/Controller/Abstract.php

```diff
@@ -85,7 +85,7 @@ public function allcartAction()
 
         $qtysString = $this->getRequest()->getParam('qty');
         if (isset($qtysString)) {
-            $qtys = array_filter(json_decode($qtysString), '\strlen');
+            $qtys = array_filter(json_decode($qtysString), fn ($tmpString) => strlen($tmpString ?? ''));
```

Comment:
Remove space after `fn`:
```suggestion
            $qtys = array_filter(json_decode($qtysString), fn($tmpString) => strlen($tmpString ?? ''));
```

Reply:
@kiatng phpcs wants the space, in a previous commit there was no space and phpcs failed https://github.com/OpenMage/magento-lts/actions/runs/9800221620/job/27061795879

Reply:
In PR #4083 we had to use a space to avoid the issue related to PHPCS error.

Reply:
Ref https://github.com/php-fig/fig-standards/blob/821ea5c5e1a30823f83e1bd56118861ded9ebbc6/accepted/PSR-2-coding-style-guide.md#6-closures:

> Control structure keywords MUST have one space after them; method and function calls MUST NOT.

Me think `fn($tmpString)` is a function call, but may be it is not.